src/Components/test/E2ETest/Infrastructure/ServerFixtures/DevHostServerFixture.cs

```diff
@@ -32,7 +38,29 @@ protected override IWebHost CreateWebHost()
                 args.Add(Environment);
             }
 
-            return DevHostServerProgram.BuildWebHost(args.ToArray());
+            return new FakeWebHost(DevHostServerProgram.BuildWebHost(args.ToArray()));
+        }
+
+        private class FakeWebHost : IWebHost
```

Comment:
Don't want to go through all the tests updating the signatures. I'll file a bug and do clean-up afterwards.